HeterogeneousCore/CUDAUtilities/src/allocate_host.cc

```diff
@@ -3,8 +3,7 @@
 #include "FWCore/Utilities/interface/Likely.h"
 
 #include "getCachingHostAllocator.h"
-
-#include <cuda/api_wrappers.h>
+#include "cuda/api_wrappers.h"
```

Comment:
If still needed, `<cuda/api_wrappers.h>` is more correct than `"cuda/api_wrappers.h"`.